packages/x11/other/fontconfig/package.mk

```diff
@@ -3,8 +3,8 @@
 # Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="fontconfig"
-PKG_VERSION="2.13.1"
-PKG_SHA256="9f0d852b39d75fc655f9f53850eb32555394f36104a044bb2b2fc9e66dbbfa7f"
+PKG_VERSION="2.13.92"
```

Comment:
please don't use devel packages unless explicitly required.

Reply:
👍 got it. Have dropped the commit.